app/views/extension/index.phtml

```diff
@@ -80,13 +80,17 @@
 </main>
 
 <?php $class = isset($this->extension) ? ' active' : ''; ?>
-<aside id="slider" class="scrollbar-thin<?= $class ?>">
-	<?php
-		if (isset($this->extension)) {
-			$this->renderHelper('extension/configure');
-		}
-	?>
+<?php $closelink = isset($this->extension) ? _url('extension', 'index') : ''; ?>
```

Comment:
I know that it is not from this PR, so feel free to leave it as it, but `isset($this->extension)` is strange or at least not optimal, and I would rather want a `!empty($this->extension)` if there is a risk that the property is not defined at all, or `$this->extension != null` or similar otherwise.

Reply:
I cannot find an option to add a `#slider` to the URL via `Minz_Request::forward()` and I do not have any idea how to add it.

The `#slider` would keep the slider open after submitting.